app/src/main/res/layout/nav_header_main.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should we add a onClickListener on this fragment that opens the prop profile fragment?

Reply:
As mentioned in my PR comment, I will delegate the linking of the current user information to my second sprint task. However no full width and height fragment will be dedicated to it. I will just replace the information displayed in the NavHeader UI.
For sure I could go a little bit further and prepare the work already here. But that PR is already big enough, and it could be useful for the other members to have it merged as soon as possible.